crates/qt-build-utils/src/lib.rs

```diff
@@ -319,6 +343,10 @@ impl QtBuild {
 
         match std::fs::read_to_string(prl_path) {
             Ok(prl) => {
+                println!("cargo:rustc-link-lib=reading {}", prl_path);
```

Comment:
i think you shouldn't have things like `reading` here (why CI is exploding) ? Or are you trying to debug what is happening at each step ?

Reply:
Yes, it was exactly that. I could push a commit cleaned of these prints.

Reply:
removed in last commit